### PR TITLE
axolotl v0.3.0 with ipex-llm

### PR DESCRIPTION
**Pull request by qiyuangong**
## Description

axolotl v0.3.0 with ipex-llm

### 1. Why the change?

LLM fine-tune with axolotl and ipex-llm

### 4. How to test?
- [ ] N/A
- [ ] Unit test
- [ ] Application test
- [x] Document test
- [ ] ...

### CODE REVIEW

File: python/llm/example/GPU/LLM-Finetuning/axolotl/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+```
+
+Ensure `use_cpu` is disable in config (`~/.cache/huggingface/accelerate/default_config.yaml`).
+
+### 3. Finetune
+
+This example shows how to run [Alpaca QLoRA finetune on Llama-2](https://github.com/artidoro/qlora) directly on Intel GPU, based on [axolotl Llama-2 qlora example](https://github.com/OpenAccess-AI-Collective/axolotl/blob/v0.3.0/examples/llama-2/qlora.yml).
+
+```
+accelerate launch finetune.py qlora.yml
```
Comment (plusbang) [inline python/llm/example/GPU/LLM-Finetuning/axolotl/README.md:48]: shall we add some descriptions about changes in yaml file? It seems we comment `optimizer: paged_adamw_32bit`

**Review by plusbang — APPROVED** [review]
other LGTM

File: python/llm/example/GPU/LLM-Finetuning/axolotl/README.md (hunk 1 above)
Comment (qiyuangong) [reply python/llm/example/GPU/LLM-Finetuning/axolotl/README.md:48]: Yes. Will add more descriptions about this change.

Comment (qiyuangong) [reply python/llm/example/GPU/LLM-Finetuning/axolotl/README.md:50]: Addressed

**Pull request merged**